### DISCUSSION TITLE
Integration with cloudflare-pages

### DISCUSSION THREAD

**Discussion by fforres** [flareact/flareact#234] [Ideas]
Hey folks, been working for a bit with cloudflare pages and really like how easy is to integrate a new development flow (using env-variables, custom build scripts, analytics, etc).

I think flareact is an awesome framework, however I'm lacking a bit the CI world. Having ssr/SSG in a cloudflare worker is 🤯❤️ , however I'm missing most of the goodies I get with cloudflare-pages.

Is there a plan to integrate the both of them? IMO, I think it'd be a perfect match.

----

Maybe there's already an integration path or guide and I'm just not aware. Could not find anything [in the docs](https://flareact.com/docs/getting-started), so if there is, I apologize in advance

---

Thanks for all the work you've been doing, All the best!

**Comment by eByte23** [comment]
Hey @fforres I'm totally with you here!
I was looking into this a little earlier in the week there need to be some changes made to support some of the same features for local development. But I believe it should be pretty easy to get flareact working with pages.

I'll try to get something working during the week if I get the time

**Reply by fforres** [reply]
That's awesome! :D

**Reply by eByte23** [reply]
So during a quick read found this [Advance Mode](https://developers.cloudflare.com/pages/platform/functions/#advanced-mode) for cloudflare pages, I believe it should be reasonably easy to setup so that it outputs the worker to here and would just need to handle the static assets correctly so I will look into it!

**Comment by traffisco** [comment]
I see cloudflare pages have Next.js out of the box. Anyone has any experience with that? Does it makes this project absolete? Your thoughts @tj-kev?

**Reply by eByte23** [reply]
I believe that is only for static html export in nextjs if I am not mistaken.
The examples provided are for [static export only](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site) there are a [range of unsupported](https://nextjs.org/docs/advanced-features/static-html-export#unsupported-features) features when using export.

**Reply by traffisco** [reply]
Then flareact still rock :)

**Comment by tj-kev** [comment]
I have not used CF Pages myself but it does look like a good target for Flareact.

**Comment by Drblessing** [comment]
+1 for this being native to cloudflare pages! Would be a great alternative to next.js on vercel, and gets the benefit of unlimited bandwith + requests.
